src/api/routes/v1/amendments.ts

```diff
@@ -222,11 +229,12 @@ async function calculateConsensus(
 }
 
 /**
- * Retrieves amendments enabled on a network.
+ * Retrieves amendments in voting on a network.
  *
  * @param id - The network id.
- * @returns List of enabled amendments.
+ * @returns List of amendments in voting.
  */
+// eslint-disable-next-line max-lines-per-function, max-statements -- Disabled for this function.
```

Comment:
nit: This comment is describing what the eslint-disable is doing, but should describe why it's necessary. (In this case, explaining why this function needs to be so long instead of being broken up)